docs/wiki/The CI environment - Publishing.md

```diff
@@ -17,11 +18,76 @@ The publishing phase concludes each module's pipeline. If all previous tests suc
 
 Besides the publishing phase's runtime, there is also the possibility to set the switch `Publish prerelease module`. This switch makes it possible to publish a prerelease version in every workflow run that is not based on `main` or `master`. This can be controlled when running the module pipeline leveraging [Module pipeline inputs](./The%20CI%20environment%20-%20Pipeline%20design#module-pipeline-inputs).
 
-> **Note**<br>
-> The `version` used for publishing any artifact is the same for all three target locations, which reduces the maintenance effort.
+> **Note**: The `version` used for publishing any artifact is the same for all three target locations, which reduces the maintenance effort.
 
 > **Note:** The orchestration options described in the [solution creation](./Solution%20creation) section work differently well with the publishing locations we offer in CARML. To help you select the best location for your use case, we provide further information [here](./Solution%20creation#publish-location-considerations) section.
 
+## Module identifiers
+
+The names of published modules differ slighly depending on the location they are published to. This is rooted in the different requirements per target location. In the following you can find the rules applied for each:
+
+<details>
+<summary>Template Specs</summary>
+
+**Actions**
+
+- Make lowercase
```

Comment:
```suggestion
- Get the relative path of the module template file parent folder
- Remove the root folder name `modules`
- Make lowercase
```

Reply:
I agree with the 2nd point, but would argue the first isn't really relevant, is it? I did not intend to describe the algorithm, but simply the transformational actions on the name.

Reply:
Sounds good, but in that case I'd suggest to remove `/deploy.bicep` from all the 'Before' examples in the file

Reply:
checking...

Reply:
done

Reply:
Added the 2nd line as `- Remove the root folder name `modules` from provided module reference` to all 3 variants